src/pages/_id/index.vue

```diff
@@ -189,6 +189,7 @@ export default {
   },
   mounted() {
     this.refreshBookmarkList();
+    this.fetchReaderIndex();
```

Comment:
These 2 promises should be in a `Promise.all` ? (Not sure about if these 2 can merge with the one below)

Reply:
Hmm, but these 2 calls do not have order dependency